projects/example-cipher-c10l-hello-world/description.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: Confidential Hello World on Cipher
+authors:
+  - Oasis Protocol Foundation: https://oasisprotocol.org
+description: |
+  Minimal <b>confidential</b> smart contract written in rust which uses the
```

Comment:
HTML in description :/

Reply:
We need some kind of formatting... Markdown would be even better, but that probably complicates stuff too much?

Reply:
With React, HTML complicates stuff, the content needs to be sanitized([DOMPurity](https://www.npmjs.com/package/dompurify)). Would be better if it would be Markdown formatted.

Reply:
Rewritten to markdown.